assets/js/pages/HostDetailsPage/HostDetailsPage.jsx

```diff
@@ -66,6 +78,7 @@ function HostDetailsPage() {
     getExportersStatus();
     refreshCatalog();
     dispatch(updateLastExecution(hostID));
+    dispatch(fetchSoftwareUpdates(hostID));
```

Comment:
this needs to be like the following
```suggestion
    dispatch(fetchSoftwareUpdates({hostId: hostID}))
```
unless we change that action

Reply:
Hmm just had a look through our codebase and we use a mix of `hostID` and `hostId` :thinking: I guess `hostID` is *more* popular though, so I'll change the action

Reply:
up to you, but the thing is we need to pass an object, not the plain id (whatever is called).
unless we do transformation in the action

```
export const createAction = createAction(
  FETCH_SOFTWARE_UPDATES,
  (hostId) => ({ payload: { hostId } })
);
```

Reply:
Ah I see what you mean. I thought I was using the action-creator in `@state/softwareUpdates`. Instead, I was calling the Saga directly :flushed: